tasks/utils.js

```diff
@@ -79,9 +78,21 @@ const contractNetworks = {
 
 const getPartnerKey = (index) => `partner${index}`;
 
+const defaultRevenueSharingAddressesPath = '../revenue-sharing-addresses.json';
+let revenueSharingAddresses = undefined;
+const getRevenueSharingAddressesFromFile = (
+    filename = defaultRevenueSharingAddressesPath
+) => {
+    if (revenueSharingAddresses === undefined) {
+        revenueSharingAddresses = require(filename);
+    }
+    return revenueSharingAddresses;
```

Comment:
What happens if revenueSharingAddresses is still undefined? Should an error be thrown?

Reply:
Yes, and that's what happens if the file is missing. This should be the default behaviour of `require` in nodejs